### PR TITLE
Newsweek

### PR DESCRIPTION
**Pull request by taiwo2**
- HTML5
- CSS3
- Bootstrap 4
- Font Awesome icon
- Media Queries

### CODE REVIEW

**Comment by descholar-ceo** [comment]
# This Pull Request is under review by [descholar-ceo](https://github.com/descholar-ceo) :eyes: :eyes: 
## Please wait a bit ...

File: stylesheet/styles.css [hunk 1]
```diff
@@ -0,0 +1,407 @@
...
+}
+
+.featured-slideshows-section h2{
+  padding-bottom: 11px;  
+  padding-left: 20px;
+}
+
+
+
+>>>>>>> d07208e8313bb37af8c74f09a53c3a2870f76f83
```
Comment (descholar-ceo) [inline stylesheet/styles.css]: Remove this

File: stylesheet/main.css [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+  margin: 0;
+  outline: 0 none;
+  padding: 0;
+  -webkit-font-smoothing: antialiased;
+}
+/* .mg-grid {
+  display: block;
+  border: 2px solid black;
+  width: 100%;
+
```
Comment (descholar-ceo) [inline stylesheet/main.css]: Remove the commented codes

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Changes Required :vertical_traffic_light: :warning: 
## Hello @taiwo2 and @saimaAnis21  :handshake: :handshake: :handshake: 

:+1: This is good to make it this far, your clone almost looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you used Sementic HTML
:+1: Your readme looks great (just one or two things to fix) but it really good I like it, keep it up :muscle: 
:+1: I see that you have put in a lot of energy to make this project, that is also a great thing worth appreciation, we are very proud of you here at [Microverse](https://microverse.org/) :gift:
#### :warning: However we need to make few changes to mark this project approved. See the requested change below :point_down: :point_down: :point_down: 
### Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes] :red_circle: 
### General
- [ ] Fix the lincense link from the readme file, it's not working
- [ ] The how to use step 2 has an error.
    :warning: It shouldn't be `cd Newsweek_replica` :point_right: It should be `cd NewsWeek-Clone`
    :warning: Remember to highlight what the user should do after doing that command `cd NewsWeek-Clone`
- [ ] Work again on the project's description from your readme file, describe your project, it makes your repo look very professional.
- [ ] As the requirements of the project say, you shouldn't be committing unrelated files to the repo, so please remove `.vscode` directory from your repo
- [ ] Remove also the `NewsWeek-Clone` directory which is in root directory of your project

### UI/UX
- [ ] The date in navbar should be formatted like the one on the sorce page 

| Your clone | Original page |
| --- | --- |
| Screenshot from 2020-12-09 20-42-52|Screenshot from 2020-12-09 20-43-50 |

:point_right: Could you see the difference :question: :question: :question: 

- [ ] The subsribe button should be formatted the same way as the source page


| Your clone | Original page |
| --- | --- |
| Screenshot from 2020-12-09 20-44-48 | Screenshot from 2020-12-09 20-45-20 |

:point_right: Could you see the difference :question: :question: :question: 

- [ ] The login link should change into user icon on mobile screens 


| Your clone | Original page |
| --- | --- |
| Screenshot from 2020-12-09 20-46-36 | Screenshot from 2020-12-09 20-47-27 |

:point_right: Could you see the difference :question: :question: :question: 

- [ ] There are some navigation links which are missing from the navbar


| Your clone | Original page |
| --- | --- |
| Screenshot from 2020-12-09 20-49-38 | Screenshot from 2020-12-09 20-48-56 |

:point_right: Could you see the difference :question: :question: :question: 

- [ ] You are missing `THE DEBATE` section 


| Your clone | Original page |
| --- | --- |
| :grey_question: :grey_question: :grey_question:  | Screenshot from 2020-12-09 20-50-20 |

- [ ] You are missing `MY TURN` section


| Your clone | Original page |
| --- | --- |
| :grey_question: :grey_question: :grey_question:  | Screenshot from 2020-12-09 20-52-44 |

- [ ] The `CULTURE & TRAVEL` should have three articles per row, yours has only two, fix that
- [ ] At the end of `Opinions` section (right most of the page) you should have added the form which has an email input field and signup button 


| Your clone | Original page |
| --- | --- |
| :grey_question: :grey_question: :grey_question:  | Screenshot from 2020-12-09 20-54-29|

- [ ] Add some spacing between social network icons at the rightmost of the footer
- [ ] In the `subscribe section` => text section, the text and button are positioned wrongly comparing to the source page, yours look centered but the original is left 


| Your clone | Original page |
| --- | --- |
| Screenshot from 2020-12-09 20-56-40 | Screenshot from 2020-12-09 20-55-59|

:point_right: Could you see the difference :question: :question: :question: 

- [ ] The news letter section has an email input field which is at the top of the signup button
- [ ] Generally your page is distorted on smaller screens (320 px)


| Your clone | Original page |
| --- | --- |
| Screenshot from 2020-12-09 20-58-50| Screenshot from 2020-12-09 20-57-50|

:point_right: Could you see the difference :question: :question: :question: 

### Codes
- [ ] Using IDs as CSS selectors is a bad practice, please use class only.
- [ ] You have used `display: flex;` in your CSS file, since this project focuses on bootstrap, please use utility classes of bootstrap to cover that
- [ ] You have very big CSS files whereas you are using bootstrap, that is because you have hardcoded most of your stylings, and you have some code repetions, please focus on a DRY css code and also some stylings use bootstrap instead of hardcoding them. For example display: block, widths, margins, paddings, responsiveness (you have a lot of media queries) => that is normally good but Bootstrap could help you to achieve responsiveness.
- [ ] Implement all changes I suggested in inline comments in your codes
- [ ] Your CSS codes has linters errors, please fix them before you submit for another review

# [Optional changes] :yellow_circle: 
- [ ] It would be very nice if you put the screenshot of your project on the readme file, it shows briefly what you have worked on, so the project might interests its visitors at the first sight
- [ ] Try to implement the polygon at the right most of the red navbar
- [ ] It should be very nice if you use SVGs in separate images folder instead of putting them directly in your HTML file.

:warning: Your commit messages should be in imperative mode or present tense, _i.e.:_ you should have written `Fix the magazine section` instead of writing `fixing the magazine section`, refer [to this article for more details on how to write a good && professional commit message](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

___
_If anything seems to be unclear to you, reachout and ask anything by commenting on this Pull Request and mention me as [descholar-ceo](https://github.com/descholar-ceo), I will receive a notification and will respond._

[GitHub](https://github.com/descholar-ceo) | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

_Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)_

**Comment by taiwo2** [comment]
please kindly give us feedback on time your reviews Time, Yours always affect our progress, the same way you did last week. we waited for more than 3 hours and before we could get your response we were a bit late to meet up with the weekly progress and our professionalism score was affected. Thanks

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

There are some issues that need to be addressed before we can move forward. Please review the list below.

## Required changes :wrench: 
### [Mandatory] :red_circle: 

> # [Required Changes] 
> ### General
- [ ]  Fix the lincense link from the readme file, it's not working. **The issue persist**
- [ ] warning Remember to highlight what the user should do after doing that command `cd NewsWeek-Clone`. **This hasn't been done***

> ### UI/UX
- [ ] The navbar don't look as expected on mobile view 320px
image
  

- [ ] Section doesn't look as expected

**CLONE**
image

**Original**
image

- [ ] Footer and navbar doesn't look as expected
**ClONE**
image
image

**Original**
image
image

- [ ] Opinion section has different styles as original
**CLONE**
image
**Original**
image

- [ ] Tablet view looks like the mobile view tablet view goes from 768px to 426px above that is desktop and below that is mobile
**CLONE**
image
image
image

**Original**
image
image
image

> ### Codes
- [ ]  Using IDs as CSS selectors is a bad practice, please use class only. **Remove IDs from CSS files*
- [ ]  You have used `display: flex;` in your CSS file, since this project focuses on bootstrap, please use utility classes of bootstrap to cover that. **Use Bootstrap class for display flex**
- [ ]  You have very big CSS files whereas you are using bootstrap, that is because you have hardcoded most of your stylings, and you have some code repetions, please focus on a DRY css code and also some stylings use bootstrap instead of hardcoding them. For example display: block, widths, margins, paddings, responsiveness (you have a lot of media queries) => that is normally good but Bootstrap could help you to achieve responsiveness. **This hasn't changed, prefer bootstrap classes**

### [Comments] :green_circle: 

- Use the browser media query inspector to observe the changes in the original page and mimic them on the clone.

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

File: stylesheet/styles.css [hunk 3]
```diff
@@ -0,0 +1,410 @@
...
+  .auth a:first-of-type {
+    padding-right: 0.7rem;
+  }
+
+  .auth a:last-of-type {
+    padding: 0 0.8rem;
+  }
+
+  .auth .icon {
+    display: block !important;
```
Comment (umairarshadbutt) [inline stylesheet/styles.css]: remove the `!important`

File: stylesheet/magazine.css [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+  color: #f72210;
+}
+
+.subscription article {
+  text-align: center;
+}
+
+.subscription,
+.subscribe article {
+  /* padding: 1.7rem 0; */
```
Comment (umairarshadbutt) [inline stylesheet/magazine.css]: remove commented CSS

File: index.html [hunk 5]
```diff
@@ -0,0 +1,1252 @@
...
+                  <p class="mb-4">By <a href="#">Eric Swalwell</a></p>
+                </div>
+              </article>
+
+              <div class="vs">
+                <div><span>VS</span></div>
+              </div>
+
+              <article class="d-flex flex-row">
+                <img class="order-2" src="images/tim-phillips.png" alt="" />
```
Comment (umairarshadbutt) [inline index.html]: Add the description in `alt` attribute

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @taiwo2  and @saimaAnis21 
Such a remarkable project you made :+1: ! Awesome README :100: !
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed (see review comment).

##

Kindly work on the suggested changes 👇.
- [ ] Remove `!important ` and commented code from css file. ([Article for support](https://speckyboy.com/good-bad-css-practices/)) ( [screenshot](https://user-images.githubusercontent.com/29689273/93404399-8997b500-f8a3-11ea-8f88-fb12b8f6ca80.png) from Pathwright)
- [ ]  You have used `display: flex;` in your CSS file, since this project focuses on bootstrap, please use utility classes of bootstrap to cover that. Use Bootstrap class for display flex.
- [ ] Add the description in `alt` attribute. see the inline comment.
##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by arslanbisharat — APPROVED** [review]
# Approved :smile: 

Hello Developer :wave:

I just reviewed your project and I am very happy to inform you that your project is approved.
Congratulations on achieving this milestone in your career. :clap:



_I am looking forward to seeing your next project._

Stay safe and give me a smile 😇

### Happy Coding
------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Comment by arslanbisharat** [comment]
Hello @saimaAnis21  and @taiwo2,
How are you doing?
I want to apologize for the inconvenience caused by me. Yesterday I claimed your PR and we had a video call about your project. But I forgot to submit a code review on GITHUB. 
I think you had to wait for many hours. I am really sorry about that!!!!

Best,
Arslan Bisharat

**Comment by taiwo2** [comment]
Fine, All we need is your support to as excel as a professional software developer, apology accepted.
    On Friday, December 11, 2020, 02:05:18 PM GMT+1, Saima Anis <notifications@github.com> wrote:  
 
 


Merged #1 into master.

**Pull request merged**